doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>In NB review, this doesn't render but it's fine in the docs. </li><li>I can't imagine what _vertically centered_ means. Is there anywhere in the examples (which I haven't read yet) which shows us the differencce that constant makes? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>So the idea is that if you don't center it then ALE(x) depends on min(x1) which is kind of an arbitrary point, by centering the plot you essentially get rid of this dependency but also get as a benefit that the ALE is zero in expectation over the marginal distro p(x), meaning that you have the nice interpretation that ALE(x)=0 means that at x you observe an effect that is average wrt p(x). I agree that it's not super clear, will think about this more, but I spent a lot of time understanding this as the paper and Molnar's chapter are not super clear.</p>